CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0]
+
 ## [1.37.0]
 ### Uncategorized
```

Comment:
Could you please categorize these changes? Most of these are either changes or additions. A few can be deleted because they are internal changes that do not affect users of this package.

Reply:
Done, pls review